src/router/httpd/visuals/filters.c

```diff
@@ -672,28 +672,28 @@ EJ_VISIBLE void ej_filter_summary_show(webs_t wp, int argc, char_t ** argv)
 
 		filter_tod_init(wp, i + 1);
 
-		websWrite(wp, "<tr class=\"center\" bgcolor=\"#CCCCCC\" >\n"
-			  "<td width=\"50\" ><font face=\"Arial\" size=\"2\" >%d.</font></td>\n"
-			  "<td width=\"200\" ><font face=\"Arial\" size=\"2\" >%s</font></td>\n"
+		websWrite(wp, "<tr class=\"table_row_bg center\" >\n"
+			  "<td width=\"50\" >%d.</td>\n"
+			  "<td width=\"200\" >%s</td>\n"
 			  "<td height=\"30\" width=\"150\" >\n"
-			  "<table width=\"150\" height=\"30\" border=\"1\" cellspacing=\"1\" bordercolor=\"#000000\" bgcolor=\"#FFFFFF\" style=\"border-collapse:collapse\" >\n" "<tr>\n", i + 1, name);
-		websWrite(wp, "<td class=\"center\" width=\"17\" bgcolor=\"%s\" style=\"border-style: solid\"><script type=\"text/javascript\">Capture(%s)</script></td>\n"
-			  "<td class=\"center\" width=\"17\" bgcolor=\"%s\" style=\"border-style: solid\"><script type=\"text/javascript\">Capture(%s)</script></td>\n"
-			  "<td class=\"center\" width=\"17\" bgcolor=\"%s\" style=\"border-style: solid\"><script type=\"text/javascript\">Capture(%s)</script></td>\n"
-			  "<td class=\"center\" width=\"17\" bgcolor=\"%s\" style=\"border-style: solid\"><script type=\"text/javascript\">Capture(%s)</script></td>\n", wp->p->tod_data_null == 0 && (wp->p->day_all == 1
+			  "<table width=\"150\" height=\"30\" border=\"1\" cellspacing=\"1\" style=\"border-collapse: collapse\" >\n" "<tr>\n", i + 1, name);
+		websWrite(wp, "<td class=\"%s center\" width=\"17\"><script type=\"text/javascript\">Capture(%s)</script></td>\n"
```

Comment:
Im unsure if it would be better to add both classes so instead of `%s center` would just be `%s` since it may get confused here otherwise.

Should work in theory either way, any insight would be welcome.

Reply:
this has been merged now and works just fine as is